modules/backend/behaviors/RelationController.php

```diff
@@ -356,6 +356,23 @@ public function initRelation($model, $field = null)
         $this->relationObject = $this->model->{$field}();
         $this->relationModel = $this->relationObject->getRelated();
 
+        // Ensure that the other side of the relationship is set on the related model
+        $parentClass = get_class($this->model);
+        $allRelationDefinitions = $this->relationModel->getRelationDefinitions();
+        $inverseRelation = null;
+        foreach ($allRelationDefinitions as $relations) {
+            foreach ($relations as $key => $value) {
+                if (strtolower($value[0]) === strtolower($parentClass)) {
+                    $inverseRelation = $key;
+                    break;
+                }
+            }
+        }
```

Comment:
@bennothommo ping

Reply:
Sorry for the extreme delay in responding to this one!

It is somewhat flimsy, but at the same time - conventionally speaking - we should be alright. I can't think of a better way right now, and even if I could, there's no telling whether it would work or not given our relation definition is completely different to Laravel's.